examples/using-subscriber/main_test.go

```diff
@@ -41,10 +41,12 @@ func initializeTest(t *testing.T) {
 func TestExampleSubscriber(t *testing.T) {
 	log := testutil.StdoutOutputForFunc(func() {
 		go main()
+
 		time.Sleep(time.Second * 1) // Giving some time to start the server
 
 		initializeTest(t)
 		time.Sleep(time.Second * 20) // Giving some time to publish events
```

Comment:
I had updated them

I assume they were rolled back as the tests were failing, I'm fine with it.

But then I would expect to see a TODO here

Maybe it could be fixed in this PR, maybe in another one

Reply:
This also will be taken care in another PR

Reply:
Should we create a separate issue?